client/src/main/java/com/vesoft/nebula/util/SslUtil.java

```diff
@@ -100,7 +103,7 @@ public static SSLSocketFactory getSSLSocketFactoryWithCA(CASignedSSLParam param)
             return context.getSocketFactory();
 
         } catch (Exception e) {
-            e.printStackTrace();
+            LOGGER.error(e.getMessage());
```

Comment:
it's better to close in `finally{}`.  In your code, if reader.readObject throws some exception, then the reader cannot be closed.